src/widgets/connector.ts

```diff
@@ -298,15 +378,15 @@ export class ConnectorView extends DOMWidgetView {
 
         const connectionTemplate = this.connectionsTemplates[key];
 
-        this.drawNewConnectionForm(connectionTemplate);
+        this.drawConnectionDetailsForm(connectionTemplate, "");
```

Comment:
we can make `connectionAlias` optional

Reply:
https://github.com/ploomber/jupysql-plugin/pull/70/commits/e48a6b3381d0cdf6900ea629819e7dccc1222c90